pubsub/api/Pubsub.Samples/PullMessagesSync.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_subscriber_sync_pull]
+
+using Google.Cloud.PubSub.V1;
+using System;
+using System.Linq;
+using System.Text;
+
+public class PullMessagesSyncSample
+{
+    public static void PullMessagesSync(string projectId,
+    string subscriptionId, bool acknowledge)
+    {
+        SubscriptionName subscriptionName = SubscriptionName.FromProjectSubscription(projectId,
+            subscriptionId);
+        SubscriberServiceApiClient subscriberClient =
+            SubscriberServiceApiClient.Create();
```

Comment:
This can be in the line above. In general, only move to the second line if the first line is too long. We don't have an 80 char limit.